docs/developer/advanced/sharing-saved-objects.asciidoc

```diff
@@ -341,12 +346,140 @@ image::images/sharing-saved-objects-step-5.png["Sharing Saved Objects ESO migrat
 
 NOTE: Reminder, don't forget to add unit tests and integration tests!
 
+[[sharing-saved-objects-phase-2]]
+=== Phase 2 Developer Flowchart
+
+This section covers switching a share-capable object type into a shareable one _or_ creating a new shareable saved object type. Refer to
+this flowchart:
+
+image::images/sharing-saved-objects-phase-2-dev-flowchart.png["Sharing Saved Objects phase 2 - developer flowchart"]
+
 [[sharing-saved-objects-step-6]]
-=== Step 6
+==== Step 6
```

Comment:
It's a little confusing to see this as step 6.  Because it's under a new phase, can numbering be restarted at 1?

Reply:
As discussed offline:

I initially used this ordering because the page is quite large and you might get lost on it, but at it's kind of nice to have all of the content on one page.

As you suggested, I tried reordering them from 1-5 and 6-8 to 1.1-1.5 and 2.1-2.3 in a0052a8e616346feac23eeff46ea814575772b57, take a look and see what you think!
Note, if we like this I'll need to change the text in the flowcharts, too.

Reply:
Honestly, I prefer it the original with step 1-5 and 6-8.